scarlet/bbox.py

```diff
@@ -45,34 +39,24 @@ def from_image(image):
         return Box(image.shape)
 
     @staticmethod
-    def from_bounds(front, back, bottom, top, left, right):
+    def from_bounds(*args):
         """Initialize a box from its bounds
 
         Parameters
         ----------
-        bottom: int
-            Minimum in the y direction.
-        top: int
-            Maximum in the y direction.
-        left: int
-            Minimum in the x direction.
-        right: int
-            Maximum in the x direction.
+        args: ints
+            Min/Max coordinate for every dimension
```

Comment:
Imagine we have a lot of dimensions, the number of arguments becomes confusing. How about asking for `N` tuples and check that `N == len(self.shape)`

Reply:
This version maintains the signature of the method, yours doesn't. I wanted to make minimal changes.

Reply:
In general I'm in favor of minimal API changes but since we just merged a major API change yesterday, making another slight tweak before we've even publicized these changes seems reasonable. As you mention in your previous comment, if we got to ever greater dimensionality like `(time, channel, height, width)` it gets increasingly difficult to make sure that we are passing the appropriate pairs of boundaries.